packages/backend/src/api/endpoints/SessionApi.ts

```diff
@@ -14,45 +14,46 @@ type QueryParams = {
 };
 
 export class SessionAPI extends AbstractAPI {
-  public getSessionList = async (queryParams?: QueryParams) =>
-    this.request<Session[]>({
+  public async getSessionList(queryParams?: QueryParams) {
+    return this.request<Session[]>({
```

Comment:
❓ Were we not returning anything here? 😨

Reply:
Messed the formatting, thanks Stef. Fixing

Reply:
Rest of the fix is valid